src/com/SerializedStamples.hpp

```diff
@@ -86,13 +86,17 @@ class SerializedStamples {
    */
   void serializeValues(const cplscheme::PtrCouplingData data);
 
+  void serializeValuesInitialization(const cplscheme::PtrCouplingData data);
```

Comment:
Could you add here and below a quick comment explaining the difference between those methods?

Reply:
Done. [9fb3bd7](https://github.com/precice/precice/pull/1671/commits/9fb3bd77800d65057d303b788d2100f467b10e19)